# patch pour corriger XAO_DATA_DIR et ne plus utiliser ABSOLUTE_APPLI_PATH
# correction attendue avec spns #11282
diff --git a/src/XAO/tests/CTestTestfileInstall.cmake b/src/XAO/tests/CTestTestfileInstall.cmake
index 443de45..9218488 100644
--- a/src/XAO/tests/CTestTestfileInstall.cmake
+++ b/src/XAO/tests/CTestTestfileInstall.cmake
@@ -20,4 +20,4 @@
 ADD_TEST(GEOM_TestXAO python ${SALOME_TEST_DRIVER} ${TIMEOUT} TestXAO)
 SET_TESTS_PROPERTIES(GEOM_TestXAO PROPERTIES
     LABELS "${COMPONENT_NAME}"
-    ENVIRONMENT "XAO_DATA_DIR=$ENV{ABSOLUTE_APPLI_PATH}/bin/salome/test/GEOM/xao/data")
+    ENVIRONMENT "XAO_DATA_DIR=$ENV{GEOM_ROOT_DIR}/bin/salome/test/xao/data")
